### DISCUSSION TITLE
App Layout

### DISCUSSION THREAD

**Discussion by Rahul-0108** [iTwin/iTwinUI#596] [Q&A]
Hi , 
I am trying to create a React App with iTwin React Components. I am not getting anything to set the Layout here : https://itwin.github.io/iTwinUI-react/?path=/story/overview--overview

Can you suggest me anything ??

**Comment by mayank99** [comment]
Could you elaborate what you mean by "I am not getting anything to set the Layout here"?

It would also help if you show us your code, highlighting any relevant parts.

**Reply by Rahul-0108** [reply]
something to set the app structure and ui zones like : https://www.telerik.com/kendo-react-ui/components/layout/ https://www.telerik.com/kendo-react-ui/components/layout/gridlayout/ https://www.telerik.com/kendo-react-ui/components/layout/splitter/

**Reply by mayank99** [reply]
Oh I see. Three things you should be aware of:
- We give you the building blocks, including some big pieces: `Header`, `SideNavigation`, `Table`, `Tile`, `Breadcrumbs`, etc.
- We recently also added a [`Surface`](https://itwin.github.io/iTwinUI-react/?path=/story/core-surface--basic&args=elevation:1) primitive for placing your content inside.
- We are currently working on some ["layout components"](https://itwin.github.io/iTwinUI-layouts/) that will help solve some of your concerns. It is not ready yet, and we recommend building your own layout (using custom CSS) until then.

**Comment by bentleyvk** [comment]
Hey @Rahul-0108, we released iTwinUI-layouts [npm](https://www.npmjs.com/package/@itwin/itwinui-layouts-react), [GitHub](https://github.com/iTwin/iTwinUI-layouts/) that should help you. Here is the demo page with code: https://itwin.github.io/iTwinUI-layouts/#/page-layout-1
